R/chop.R

```diff
@@ -165,13 +165,19 @@ df_unchop <- function(x, ..., ptype = NULL, keep_empty = FALSE, error_call = cal
     return(out)
   }
 
+  x_ptypes <- imap(x, function(col, name) {
```

Comment:
I think I prefer `map2(x, names(x), ...)` here since it makes it even more clear that you expect `x` to be named at this point.

Reply:
I seem to almost always prefer `map2()` over `imap()`. I only did this because it seemed like we use `imap()` elsewhere too and I thought you preferred it. I'll change it here because I also want `map2()`